package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Common objects shared to various Kuzzle components and plugins",
   "main": "./index.js",
   "author": "The Kuzzle Team <support@kuzzle.io>",
-  "version": "3.0.14",
```

Comment:
You only have to update the version if this is a hotfix, but the destination branch is `3-dev`

Reply:
Hmm it's updated automatically